### PR TITLE
Commit with files

### PR DESCRIPTION
**Pull request by OleksiiKlymenko**

### CODE REVIEW

File: src/main/java/com/example/bookstore/BookStoreApplication.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                kobzar.setAuthor("Shevchenko");
+                kobzar.setIsbn("something");
+
+                bookService.save(kobzar);
+
+                System.out.println(bookService.findAll());
+            }
+        };
+    }
+
```
Comment (kozhukhovsky) [inline src/main/java/com/example/bookstore/BookStoreApplication.java]:
```suggestion
```

File: src/test/java/com/example/bookstore/BookStoreApplicationTests.java
Comment (kozhukhovsky) [inline src/test/java/com/example/bookstore/BookStoreApplicationTests.java:1]: You can delete this class

**Review by kozhukhovsky — APPROVED** [review]

**Comment by OleksiiKlymenko** [comment]
I made mistake and did second home work in the same branch. So here you can see second home work as a second commit with message "Added controller, service layer, and created DTO's"

Please, can you check this home work in this branch? Then I will merge them and will make home work in an another branch?

File: src/main/java/com/example/bookstore/service/BookServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import com.example.bookstore.dto.CreateBookRequestDto;
+import com.example.bookstore.mapper.BookMapper;
+import com.example.bookstore.model.Book;
+import com.example.bookstore.repository.BookRepository;
+import jakarta.persistence.EntityNotFoundException;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+
+//@RequiredArgsConstructor
```
Comment (Ivan95kos) [inline src/main/java/com/example/bookstore/service/BookServiceImpl.java:13]: remove it
```suggestion
```

**Review by Ivan95kos — APPROVED** [review]
Great job, with minor comments

**Pull request merged**